docs/UserGuide.md

```diff
@@ -194,7 +207,10 @@ Examples:
 
 ### Finding animals by their fields: `find`
 
-Finds and lists all animals in the `ZooKeep` book whose fields contain any of the specified argument keywords (at least 1).
+You may find yourself in situations where you need to retrieve a specific group of animals from the `ZooKeep` book. For
+example, you may need to find animals that have the same feed times or animals that have similar medical conditions.
```

Comment:
Just a suggestion: find animals with same feed times or similar medical conditions

Reply:
Fixed this and all other issues above